win32/Lib/win32serviceutil.py

```diff
@@ -47,7 +42,7 @@ def LocatePythonServiceExe(exe=None):
     # Even if that file already exists, we copy the one installed by pywin32
     # in-case it was upgraded.
     # pywin32 installed it next to win32service.pyd (but we can't run it from there)
-    maybe = os.path.join(os.path.dirname(win32service.__file__), exe)
+    maybe = os.path.join(os.path.dirname(win32service.__file__), default_service_exe)
```

Comment:
and here we'd be trying to use default_service_exe even if a different value was supplied above, which seems wrong.